setup.py

```diff
@@ -0,0 +1,33 @@
+from setuptools import setup, find_packages
+
+
+def readme():
+    with open('README.rst') as f:
+        return f.read()
+
+setup(name='deep_qa',
+      version='0.1',
+      description='Using deep learning to answer Aristo\'s science questions',
```

Comment:
I would change this description.  Something like "NLP library built on top of Keras, focusing on question answering."